tests/foreman/cli/test_contentaccess.py

```diff
@@ -134,19 +136,21 @@ def setUpClass(cls):
     @run_only_on('sat')
     @tier2
     def test_positive_list_installable_updates(self):
-        """Run `hammer host errata list` and assert all updates are listed on
-        packages tab updates and not only those for attached subscriptions.
+        """Ensure packages applicability is functioning properly.
```

Comment:
why does test case description is changed so much?

Reply:
This test repeat the same doc as the next test, but it was intended to check package applicability

Reply:
plus, this a pure CLI test, as we have the same tests in UI